frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -77,6 +77,21 @@ local symbol_prefix = {
         mem_usage = "",
         wifi_status = "",
         wifi_status_off = "",
+    },
+    compact_items = {
+        time = nil,
+        pages_left = BD.mirroredUILayout() and "<" or ">",
```

Comment:
This probably isn't the desired character, but the one we quickly type because we don't know how to type a more suitable UTF-8 symbol.

Reply:
I was going for something similar to the right/left pointing arrows as in icons. I can replace them with the icon character if preferred.

Reply:
I don't know what you're referring to precisely, but I'm talking about something like the difference between greater than and a chevron (like `⟩`).

Reply:
Yes, no need for icons, there might be more accurate unicode symbols, the classic < > might not be that symetric in some fonts (or dunno, may be they are ? :)

Reply:
https://shapecatcher.com/ draw > or <, and see what looks like it :) 

But I guess what you might want are the buddies of the "«" or "»" we use, but single quote.
U+203A https://www.fileformat.info/info/unicode/char/203a/index.htm